app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -78,6 +78,23 @@ def toggle_notifications
       redirect_to support_interface_provider_user_path(provider_user)
     end
 
+    def impersonate
+      @provider_user = ProviderUser.find(params[:provider_user_id])
+      dfe_sign_in_user.begin_impersonate! session, @provider_user
+      flash[:success] = "Impersonating #{@provider_user.email_address}"
```

Comment:
I think this message should match what we say when signing in as a candidate.

<img width="1000" alt="Screenshot 2021-01-04 at 12 07 24" src="https://user-images.githubusercontent.com/813383/103533725-8988e880-4e85-11eb-8a8a-f335d8067f1e.png">

Reply:
Thanks for reviewing! I made this change but have since removed the flash message entirely because of tweaks to this flow for support users. Which now is:

![image](https://user-images.githubusercontent.com/107591/103892807-1e891d00-50e4-11eb-8119-7f4299496180.png)
![image](https://user-images.githubusercontent.com/107591/103892822-25179480-50e4-11eb-9a91-d44c98fc82fa.png)
![image](https://user-images.githubusercontent.com/107591/103892839-29dc4880-50e4-11eb-8981-36995d136f87.png)

Reply:
I've added a masks emoji in support, it will be useful as many of us have also created real provider users with the same DfE sign in uid as our support user on various environments.

Reply:
![image](https://user-images.githubusercontent.com/107591/103897035-d6b9c400-50ea-11eb-9684-3028905fb725.png)